### ISSUE TITLE
Results are different when using the  marginal_smooths on the same machine

### ISSUE THREAD

**Issue by Spatial-R** [paul-buerkner/brms#800]
Hello, thanks for your development the brms packages. Recently, i have used the brms package to fit the non-linear effects  of  two predictors  on the response.  However, i come across a problem and hope you can help me. 

When i finished the model fitting, i used the `marginal_smooths` function to see the results. I saved the model fitting (brmsfit), closed the console and opened another RStudio in the same machine to run the `marginal_smooths` function again, the  results were quite different.  Is there any problem to my problem?

`
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17134)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.936  LC_CTYPE=Chinese (Simplified)_China.936    LC_MONETARY=Chinese (Simplified)_China.936
[4] LC_NUMERIC=C                               LC_TIME=Chinese (Simplified)_China.936    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] brms_2.10.0 Rcpp_1.0.3 

loaded via a namespace (and not attached):
 [1] lattice_0.20-38      prettyunits_1.0.2    ps_1.3.0             zoo_1.8-6            gtools_3.8.1         assertthat_0.2.1     digest_0.6.23       
 [8] mime_0.7             R6_2.4.1             plyr_1.8.4           backports_1.1.5      ggridges_0.5.1       stats4_3.6.1         coda_0.19-3         
[15] ggplot2_3.2.1        colourpicker_1.0     pillar_1.4.2         rlang_0.4.2          lazyeval_0.2.2       rstudioapi_0.10      miniUI_0.1.1.1      
[22] callr_3.3.2          Matrix_1.2-17        DT_0.10              shinythemes_1.1.2    shinyjs_1.0          stringr_1.4.0        htmlwidgets_1.5.1   
[29] loo_2.1.0            igraph_1.2.4.1       munsell_0.5.0        shiny_1.4.0          rstan_2.19.2         compiler_3.6.1       httpuv_1.5.2        
[36] pkgconfig_2.0.3      base64enc_0.1-3      pkgbuild_1.0.6       rstantools_2.0.0     htmltools_0.4.0      tidyselect_0.2.5     tibble_2.1.3        
[43] gridExtra_2.3        threejs_0.3.1        matrixStats_0.55.0   crayon_1.3.4         dplyr_0.8.3          later_1.0.0          grid_3.6.1          
[50] nlme_3.1-140         xtable_1.8-4         gtable_0.3.0         lifecycle_0.1.0      magrittr_1.5         StanHeaders_2.19.0   scales_1.1.0        
[57] cli_1.1.0            stringi_1.4.3        reshape2_1.4.3       promises_1.1.0       dygraphs_1.1.1.6     xts_0.11-2           tools_3.6.1         
[64] glue_1.3.1           shinystan_2.5.0      markdown_1.1         purrr_0.3.3          crosstalk_1.0.0      processx_3.4.1       rsconnect_0.8.15    
[71] abind_1.4-5          parallel_3.6.1       fastmap_1.0.1        inline_0.3.15        colorspace_1.4-1     bridgesampling_0.7-2 bayesplot_1.7.0     
[78] Brobdingnag_1.2-6   `

**Comment by paul-buerkner** [comment]
I will need a minimal reproducible example in order to say more about this potential problem. It could be that you updated to a new version of brms but even then the results should not change.

**Comment by Spatial-R** [comment]
Thanks for your reply. The reproducible example is [here](https://github.com/Spatial-R/Dynamic-Model-for-Infectious-Disease/tree/master/Example). The link includes the analysis codes and the dataset.  It is very strange that the results from the `marginal_smooths` function is consistent  in two Linux system, but fail to keep consistent in two different system (I fitted the model in a Linux machine and re-run the  `marginal_smooths` function in a Window machine)  and two Window machines.

**Comment by paul-buerkner** [comment]
I am sorry, but this model/data is too big to run it as a minimal reproducible example, which should take minutes at most to run. I tried it with a subset of your data on a windows machine and I get the same results within the session and after loading the model in a new session. There is also no reason why this should be any different with regard to the results of marginal_smooths. Closing this issue as this is (to the best of my understandig) not a bug in brms.
